app/src/main/res/layout/fragment_dashboard.xml

```diff
@@ -84,6 +84,7 @@
                     android:layout_height="wrap_content"
                     android:layout_marginBottom="8dp"
                     android:layout_marginTop="8dp"
+                    android:paddingBottom="80dp"
```

Comment:
Tohle je tady proc? Odsouva to ten subtitle

Reply:
Chyba, sorry. Mělo to řešit tohle: https://trello.com/c/u5DHfGH1/114-graficke-nedostatky-aplikace-mensi-zarizeni-and-010174, ale ten problém jsem pochopil špatně. Jde o to, že to průhledné tlačítko nejde číst přes text. Mám prostě to tlačítko změnit, aby nemělo průhledné pozadí?

Reply:
Jo asi dava smysl predelat backgroundTint ve styles pro outlinedButton na ?colorSurface

Reply:
Nebo tak neco..aby to fungovalo i pro dark mode